app/src/main/java/pl/valueadd/restcountries/presentation/main/countries/list/CountryListFragment.kt

```diff
@@ -48,7 +73,42 @@ class CountryListFragment : BaseMVPFragment<CountryListView, CountryListPresente
             .start(fragment)
     }
 
-    private fun setupView() {
+    override fun onQueryTextSubmit(query: String?): Boolean =
+        onQueryTextChange(query)
+
+    override fun onQueryTextChange(newText: String?): Boolean {
+        presenter.onQueryChanged(newText ?: EMPTY)
+
+        return false
```

Comment:
regarding documentation this should return `true` https://developer.android.com/reference/android/widget/SearchView.OnQueryTextListener.html#onQueryTextChange(java.lang.String)